index.js

```diff
@@ -138,6 +138,7 @@ function bootstrap(options) {
 
   app.use(userMiddleware);
   app.use(hofMiddleware.cookies());
+  app.use(require('hof-middleware-markdown')());
```

Comment:
Yeah, no-one ever went wrong putting all their `require`s at the top of the file.

The key is to always have them at the root of the module though. If you do that then you automatically resolve the full dependency tree in the first pass of the event loop, and you know that once your app has started then it won't later fail due to a missing module. And if it is going to fail then it will fail immediately on start-up.

By putting a require inside a function you open up the possibility that it might get called later, and error in the future. So keeping them all at the top is a nice pair of bullet-proof socks.